astroquery/exoplanets/exoplanet_archive.py

```diff
@@ -0,0 +1,82 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import json
+import os
+from astropy.utils.data import download_file
+from astropy.io import ascii
+import astropy.units as u
+
+__all__ = ['query_exoplanet_archive_catalog']
+
+EXOPLANETS_CSV_URL = ('http://exoplanetarchive.ipac.caltech.edu/cgi-bin/'
+                      'nstedAPI/nph-nstedAPI?table=exoplanets')
+EXOPLANETS_TABLE = None
+PARAM_UNITS = None
+TIME_ATTRS = {'rowupdate': 'iso'}
+BOOL_ATTRS = ('pl_kepflag', 'pl_ttvflag', 'pl_k2flag', 'st_massblend',
+              'st_optmagblend', 'st_radblend', 'st_teffblend')
+
+
+def exoplanet_archive_units():
+    """
+    Dictionary of exoplanet parameters and their associated units.
+    """
+    global PARAM_UNITS
```

Comment:
Yes, that's about right.  Something like:
```
class ExoplanetTable(object)
    @property
    def PARAM_UNITS(self):
        if hasattr(self, '_param_units'):
              return self._param_units
        else: 
              # populate self._param_units
```
is a common pattern I use.  I don't like the all-caps function name in this specific case, but something along these lines should work...